src/frontend/src/app/pages/profile-edit/reviewer-personal-info-form/reviewer-personal-info-form.component.ts

```diff
@@ -120,26 +118,26 @@ export interface ReviewerProfileForm {
       </app-key-value-grid>
 
       <div class="flex items-center justify-end">
+        @if (profileForm.invalid) {
+          <div class="text-error pr-5 text-sm md:pr-10">
+            Fix the validation errors
```

Comment:
_"Uh-oh! There are some errors in your form. Please fix them and try again."_ or similar ;)